app/src/main/java/com/quarkworks/apartmentgroceries/auth/UserActivity.java

```diff
@@ -21,101 +24,41 @@
 
 import java.io.IOException;
 
+import io.realm.Realm;
+import io.realm.RealmBaseAdapter;
+import io.realm.RealmQuery;
+import io.realm.RealmResults;
+
 public class UserActivity extends AppCompatActivity {
 
-    private static final String LOG_TAG = UserActivity.class.getSimpleName();
+    private static final String TAG = UserActivity.class.getSimpleName();
 
-    private String sessionToken;
-    private TextView usernameTextView;
-    private TextView phoneTextView;
-    private EditText phoneEditText;
+    private RealmResults<RUser> users;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.user_activity);
 
-        usernameTextView = (TextView) findViewById(R.id.user_activity_username_id);
-        phoneTextView = (TextView) findViewById(R.id.user_activity_phone_id);
-        phoneEditText = (EditText) findViewById(R.id.user_activity_edit_text_phone_id);
-
-        SharedPreferences sharedPreferences = getSharedPreferences("login", 0);
+        SyncUser.getAllUser();
 
-        sessionToken = sharedPreferences.getString("sessionToken", null);
-        Toast.makeText(this, "sessionToken:" + sessionToken, Toast.LENGTH_LONG).show();
+        Realm realm = Realm.getInstance(this);
 
-        String username = sharedPreferences.getString("username", null);
-        String phone = sharedPreferences.getString("phone", null);
-
-        usernameTextView.setText(username);
-        phoneTextView.setText(phone);
-    }
+        RealmQuery<RUser> query = realm.where(RUser.class);
```

Comment:
You can also do realm.where(RUser.class).findAll();

You could even do DataStore.getInstance().getRealm().were(RUser.class).findAll(); if you wanted to.